### PR TITLE
Improve Vietnamese translation

### PR DESCRIPTION
**Pull request by nghiahsgs**
**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by nghiahsgs** [comment]
Ref: https://github.com/lablup/backend.ai/pull/2855#issuecomment-2365479263

File: resources/i18n/vi.json [hunk 1]
```diff
@@ -163,7 +163,7 @@
       "DescSingleNode": " <p> Khi chạy một phiên, nút được quản lý và các nút công nhân được đặt trên một nút vật lý hoặc máy ảo duy nhất. </p>",
       "DescMultiNode": " <p> Khi chạy một phiên, một nút được quản lý và một hoặc nhiều nút công nhân được chia thành nhiều nút vật lý hoặc máy ảo. </p>",
       "CPU": "CPU",
-      "Memory": "Ký ức",
+      "Memory": "Bộ nhớ",
```
Comment (rapsealk) [inline resources/i18n/vi.json:166]: Both "Ký ức" and "Bộ nhớ" are translated as "memory" in English. Therefore, I inquired with ChatGPT about the distinctions between these two terms.

In Vietnamese, "Ký ức" and "Bộ nhớ" have distinct meanings:

Ký ức: This term refers to memory in the psychological or emotional sense, often related to personal experiences, recollections, or memories that individuals hold.

Bộ nhớ: This term specifically refers to memory in a technical context, particularly in computing. It denotes the hardware or storage used in computers and devices to store data temporarily or permanently.

So, for computer memory, "Bộ nhớ" is the correct term to use.

**Comment by rapsealk** [comment]
LGTM. Could you please review it once more, @agatha197? Thank you.

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**